synedrion/src/paillier/keys.rs

```diff
@@ -12,20 +11,12 @@ use crate::uint::{
     RandomPrimeWithRng, Retrieve, Signed, UintLike, UintModLike,
 };
 
-#[derive(Clone, Serialize, Deserialize, ZeroizeOnDrop)]
+#[derive(Clone, Debug, Serialize, Deserialize, ZeroizeOnDrop)]
```

Comment:
Does it somehow obscure the secret values in the output, the way the previous implementation did?

Reply:
Yeah, it doesn't log the inner value. See the [implementation on `SecretBox` here](https://docs.rs/secrecy/0.9.0-pre.0/src/secrecy/lib.rs.html#143).

Reply:
But `p` and `q` are not `SecretBox`-ed.

Reply:
Oh sorry, you're right. I thought they were `SecretBox`-ed for some reason. I've added a manual `Debug` implementation that matches the `DebugSecret` one.

One other option here would be to `SecretBox` `p` and `q`, but it would add bit of noise of this PR. Wdyt?